src/demo-app/demo-app/demo-app.scss

```diff
@@ -1,3 +1,19 @@
+// The default indigo-pink theme will load through the index file. This makes the SCSS build faster
+// since Sass doesn't need to build a "prebuilt" theme again.
+@import '../../lib/core/theming/all-theme';
```

Comment:
Not sure how to make this work, the demo app cannot reach into the lib directory directly. Would be best to have 

```
@import '../@angular/material/core/theming/all-theme';
```

but the sass compiler expects this to be present during build but it is only there after build. Any other way to import this file dynamically?